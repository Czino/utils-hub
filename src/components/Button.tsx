export const Button = ({ className, children, ...props }: React.ComponentProps<'button'>) => (
  <button className={`p-4 uppercase bg-orange-600 text-yellow-100 ${className}`} {...props}>
    {children}
  </button>
)
