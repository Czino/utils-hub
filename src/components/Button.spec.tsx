import { render } from '@testing-library/react'
import { Button } from './Button'

describe('Button', () => {
  it('should render Button with styles', () => {
    const { asFragment } = render(
      <Button className="mt-4" onClick={jest.fn()}>
        Test
      </Button>
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
