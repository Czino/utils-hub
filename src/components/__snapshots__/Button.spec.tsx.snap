// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render Button with styles 1`] = `
<DocumentFragment>
  <button
    class="p-4 uppercase bg-orange-600 text-yellow-100 mt-4"
  >
    Test
  </button>
</DocumentFragment>
`;
